dartagnan/src/main/java/com/dat3m/dartagnan/parsers/program/exception/UnrollException.java

```diff
@@ -0,0 +1,8 @@
+package com.dat3m.dartagnan.parsers.program.exception;
+
+public class UnrollException extends IllegalStateException {
```

Comment:
I think this might be superfluous. I haven't checked all usages of it though but I think it might be too specific again (`IllegalStateException` might be sufficient on its own).